hub/util/no_view.py

```diff
@@ -0,0 +1,18 @@
+from hub.util.exceptions import InvalidOperationError
+from typing import Callable
+from functools import wraps
+import hub
+
+
+def no_view(callable: Callable):
+    @wraps(callable)
+    def inner(x, *args, **kwargs):
+        func = callable.__name__
```

Comment:
No need for extra variable `func` which is unused unless there's an error.  Replace `func` in line 13 with `callable.__name__` directly or move this to inside the if.